e2e/cypress/utils/index.js

```diff
@@ -47,6 +48,15 @@ export function getMessageMenusPayload({dataSource, options} = {}) {
     return data;
 }
 
+export function getMessageButtonsPayload() {
+    const data = messageButtons;
+    const callbackUrl = Cypress.env().webhookBaseUrl + '/message_buttons';
+    data.attachments[0].actions[0].integration.url = callbackUrl;
+    data.attachments[0].actions[1].integration.url = callbackUrl;
```

Comment:
I would like to see this function to not use fixed indices but instead set the url for all attachments and actions.

Reply:
Sure, though it's just a good to have but not necessary since it's based on fixed and not dynamic JSON values.

Reply:
I'm 1/5 that someone will stumble across this at some point and will not set the URL correctly.

But not a blocker.